UPGRADE.md

```diff
@@ -37,6 +37,104 @@ are terminated. In this way, Kong will serve new requests via the new
 configuration, without dropping existing in-flight connections.
 
 
+
+## Upgrade to `1.3`
+
+#### 1. Breaking Changes
+
+##### Dependencies
+
+If you are using the provided binary packages, all necessary dependencies
+are bundled. If you are building your dependencies by hand, you should
+be aware of the following changes:
+
+- The required OpenResty version has been bumped to 1.15.8.1. Kong's
+  [openresty-patches](https://github.com/kong/openresty-patches) and
+  the newly introduced [lua-kong-nginx-module](https://github.com/Kong/lua-kong-nginx-module)
+  are **required** for Kong to work properly.
+- The minimum required OpenSSL version is 1.1.1 (same with 1.2.x).
+  If you are building by hand, make sure all dependencies,
+  including LuaRocks modules, are compiled using the same OpenSSL version.
+
+##### Core
+
+- :warning: Fixed a bug in router that when wildcard host name and plain host name
```

Comment:
These issue description have the same problems as the ones in the changelog. Please them over after fixes are done.